views/index.erb

```diff
@@ -1,32 +1,5 @@
 <main class="content">
   <form action="/portal" method="POST">
-
-    <fieldset class="form-info">
-      <h2 class="form-info__title">User Settings</h2>
-
-      <div class="form-field">
-        <div class="input-field">
-          <label class="label" for="name">Full name</label>
-          <input class="input" id="name" type="text" value="<%= @current_user[:name] %>" placeholder="John Smith" />
```

Comment:
Should be able to remove the entire `@current_user` instance variable from `app.rb`